schema-source/RSxxSub.schema.yaml

```diff
@@ -0,0 +1,219 @@
+      Data Type: "String"
+    model_number:
+      Description: "Model number"
+      Data Type: "Pattern"
+      Notes: "Pattern shall match all model numbers that can be represented by the representation"
+
+Performance:
+  Object Type: "Data Group"
+  Data Elements:
+    thickness:
+      Description: "The total thickness of the glazing."
+      Data Type: "Numeric"
+      Units: "m"
+      Required: True
+    conductivity:
+      Description: "Overall conductivity of the glazing."
+      Data Type: "Numeric"
+      Units: "W/m-K"
+      Required: True
+    front_side_infrared_emissivity:
+      Description: "Front Side Infrared Emissivity"
+      Data Type: "Numeric"
+      Units: "-"
+      Required: True
+      Constraints: [">=0.0", "<=1.0"]
+    back_side_infrared_emissivity:
+      Description: "Back Side Infrared Emissivity"
+      Data Type: "Numeric"
+      Units: "-"
+      Required: False
+      Constraints: [">=0.0", "<=1.0"]
+      Notes: "When this data element is missing, the value is assumed to be the same as the front_side_infrared_emissivity."
+    performance_map_optical:
+      Description: "Data group describing optical performance over a range of conditions. A list is provided that corresponds to the lists of wavelength_integration, is_incidence_hemispherical, and is_emergence_hemispherical."
+      Data Type: "[{PerformanceMapOptical}]"
+      Required: True
+    is_incidence_hemispherical:
+      Description: "Indicates that the incidence is hemispherical"
+      Data Type: "[Boolean]"
+      Notes: "True if light arrives from all directions of the hemisphere on the sample surface. If True incidence_angle_polar and incidence_angle_azimuth should not be populated. If False they are expected to be populated."
+      Required: True
+    emergence_direction:
+      Description: "Direction of emergence"
+      Data Type: "[{EmergenceDirectionOptions}]"
+      Notes: "Diffuse means all directions with an emergence angle greater than 10 degrees. Hemispherical means all directions to the hemisphere above the sample surface. Both are used when the emergence in many directions is treated as a sum. SPECIFIC_DIRECTION indicates that specific emergence_angle_polar and emergence_angle_azimuth will be used." 
+      Required: True
+    wavelength_integration:
+      Description: "Spectrum over which the integral result was measured or calculated. `ultraviolet`, `visible`, `solar` and `infrared` indicate the range of the wavelengths used for the integral values. Details about the spectrum are provided by the method which has been applied to generate the data set. If neither `ultraviolet`, `visible`, `solar` and `infrared` describe the spectrum correctly, `other` indicates that all information about the spectrum is provided by the applied method. If SPECIFIC_WAVELENGTHS is used then GridVariablesOptical.wavelength is used and should be populated."
+      Data Type: "[{WavelengthIntegrationOptions}]"
+      Required: True
```

Comment:
Suggest removing since the map can indicate average properties using large wavelength spans.